_security/access-control/permissions.md

```diff
@@ -87,6 +100,11 @@ These permissions are for the cluster and can't be applied granularly. For examp
 - cluster:admin/opendistro/ad/result/search
 - cluster:admin/opendistro/ad/result/topAnomalies
 - cluster:admin/opendistro/ad/tasks/search
+
+### Alerting permissions
+
+See [Alerting API]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/api/).
+
 - cluster:admin/opendistro/alerting/alerts/ack (acknowledge)
```

Comment:
Is `(acknowledge)` part of the permission? If not it should be removed from the line.